packages/cli/package.json

```
@@ -19,6 +19,8 @@
     "coverage": "codecov",
     "build": "node scripts/build.mjs",
     "dev": "ts-node ./src/index.ts",
+    "vercel": "pnpm dev",
+    "vc": "pnpm dev",
```

Let's just copy pasta IMO:

```suggestion
    "vercel": "ts-node ./src/index.ts",
    "vc": "ts-node ./src/index.ts",
```

---

Wouldn't this open up the possibility of these diverging in some future?

---

oops, automerged. I can do a fixup pr if you like.

---

No biggie.